Update 'animation-iteration-event-destroy-renderer.html' to use 'testharness'

Ahmad-S792 opened the pull request:
#### a75d4f229c6f83b09c4dd8ae9f1039bee4c46843
<pre>
Update &apos;animation-iteration-event-destroy-renderer.html&apos; to use &apos;testharness&apos;

<a href="https://bugs.webkit.org/show_bug.cgi?id=266063">https://bugs.webkit.org/show_bug.cgi?id=266063</a>

Reviewed by Antoine Quint.

Merge (and update): <a href="https://chromium.googlesource.com/chromium/src.git/+/13fe1b8357c3e67f72136207efdafdaf8a103100">https://chromium.googlesource.com/chromium/src.git/+/13fe1b8357c3e67f72136207efdafdaf8a103100</a>

This PR imports the following tests from Blink / Chromium source with slight changes
(e.g., adding DOCTYPE, `html` tag, unprefixed CSS etc.) to make it &apos;testharness&apos; based and more resilient.

* LayoutTests/animations/animation-iteration-event-destroy-renderer.html: Rebaelined
* LayoutTests/animations/animation-iteration-event-destroy-renderer-expected.txt: Rebaselined

Canonical link: <a href="https://commits.webkit.org/271726@main">https://commits.webkit.org/271726@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/85c25ece2fbaee1b4ef1f5e7b9d66240889827f5

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/29475 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/8142 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/30800 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/32020 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/26722 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/10284 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/5433 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/26722 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/29748 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/6795 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/25185 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/5816 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/5950 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/26268 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/33361 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/26906 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/26689 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/32170 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [❌ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/5907 "Failed to checkout and rebase branch from PR 21481") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/4093 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/29941 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/7631 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/26069 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/6441 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/3788 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/45/builds/6430 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

graouts reviewed (approved):
Fine to land as-is, but I think we should remove all the prefixes (`-webkit-` for CSS properties and `webkit` for JS properties and DOM event names).

Ahmad-S792 committed:
Update 'animation-iteration-event-destroy-renderer.html' to use 'testharness'

https://bugs.webkit.org/show_bug.cgi?id=266063

Reviewed by Antoine Quint.

Merge (and update): https://chromium.googlesource.com/chromium/src.git/+/13fe1b8357c3e67f72136207efdafdaf8a103100

This PR imports the following tests from Blink / Chromium source with slight changes
(e.g., adding DOCTYPE, `html` tag, unprefixed CSS etc.) to make it 'testharness' based and more resilient.

* LayoutTests/animations/animation-iteration-event-destroy-renderer.html: Rebaelined
* LayoutTests/animations/animation-iteration-event-destroy-renderer-expected.txt: Rebaselined

Canonical link: https://commits.webkit.org/271726@main
